exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -237,33 +260,32 @@ func (b *Bitfinex) UpdateTradablePairs(forceUpdate bool) error {
 		return err
 	}
 
-	return b.UpdatePairs(currency.NewPairsFromStrings(pairs), asset.Spot, false, forceUpdate)
+	return b.UpdatePairs(currency.NewPairsFromStrings(pairs),
```

Comment:
margin functionality returns "[BTC JPY LEO XTZ ZEC EDO BTG NEO OMG BSV ETP ETH XAUT LTC EOS ETC UST XLM ZRX USD DSH GBP XRP EUR SAN BAB IOT XMR]"

UpdatePairs requires a currency.Pairs value and some things are not going to work. Might need to add in an interface. 

This will need to be addressed in a another pass though. Will add it to trello.

Reply:
I think I put in the wrong function, sorry! I'll add a new comment on the relevant section